packages/@aws-cdk-testing/framework-integ/test/aws-ecs/test/integ.cluster-bottlerocket-nvidia-ami.ts

```diff
@@ -0,0 +1,26 @@
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as cdk from 'aws-cdk-lib';
+import * as ecs from 'aws-cdk-lib/aws-ecs';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-ecs-integ-bottlerocket-nvidia-ami');
+
+const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 2, restrictDefaultSecurityGroup: false });
+
+const cluster = new ecs.Cluster(stack, 'Cluster', {
+  vpc,
+});
+cluster.addCapacity('bottlerocket-asg', {
+  minCapacity: 2,
+  instanceType: new ec2.InstanceType('c5.large'),
```

Comment:
Certainly! I'll try it later.

Reply:
I was asking because from the Console you can deploy only with certain GPU-based instance types and it seemed strange that `c5.large` would deploy. Not sure if it's possible/worth validating this.

Reply:
As you mentioned, this behavior does seem odd. However, since the integration test itself was successful, does this mean that deployment is possible via CloudFormation? Just to be sure, I have switched to integration testing with instances that have GPUs. Please let me know if further verification is needed.

Reply:
I guess that CloudFormation correctly deploys the ASG but the instances will fail to launch if the instance type is incompatible with the provided AMI.
We may want to leave it to the user to provide a valid configuration (unless maintainers think otherwise).